src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const temp = this[i];
+
+          this[i] = this[j];
+          this[j] = temp;
+        }
+      }
+    }
```

Comment:
The implementation uses a bubble sort algorithm, which is correct for this task. However, keep in mind that bubble sort has a worst-case and average complexity of O(n^2), which makes it inefficient on large lists. For practice, you might want to explore more efficient sorting algorithms like quicksort or mergesort in the future.